### PR TITLE
add missing info

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/OfficeDocs-Support/issues/1521

### CODE REVIEW

File: SharePoint/SharePointOnline/lists-and-libraries/relative-url-across-site-collection-not-supported.md [hunk 1]
```diff
@@ -17,12 +17,15 @@ appliesto:
 
 # Using relative URLs for content types across site collections is not supported
 
-In Microsoft SharePoint Online, when you specify the **Document Template** for a site content type by using a URL of an existing document template, make sure that you use the URL of an existing site in the same site collection. A relative URL to a content type is not supported in the content type syndication scenario.
+In Microsoft SharePoint Online, when you specify the **Document Template** for a site content type by using a URL of an existing document template, make sure that you use the URL of an existing site in the same site collection. A relative URL (valid also for absolute URL) to a content type is not supported in the content type syndication scenario.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/lists-and-libraries/relative-url-across-site-collection-not-supported.md:20]:
```suggestion
In Microsoft SharePoint Online, when you specify the **Document Template** for a site content type by using a URL of an existing document template, make sure that you use the URL of an existing site in the same site collection. A relative URL (also valid for an absolute URL) to a content type is not supported in the content type syndication scenario.
```

File: SharePoint/SharePointOnline/lists-and-libraries/relative-url-across-site-collection-not-supported.md [hunk 2]
```diff
@@ -17,12 +17,15 @@ appliesto:
...
 
 Specifically, under **Site Content Type** > **Advanced Settings** > **Document Template**, the URL (if any) has to be a URL of an existing content type on the same site collection.
 
 ![Enter the URL for Document Template in Site Content Type: Advanced Settings page](./media/relative-url-across-site-collection-not-supported/enter-url.png)
 
+> [!NOTE]
+> Issue affects also SharePoint Server on-premises.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/lists-and-libraries/relative-url-across-site-collection-not-supported.md:27]:
```suggestion
> This issue also affects SharePoint Server on-premises.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @VLG17, kindly commit the suggested changes. Many thanks.

**Comment by simonxjx** [comment]
Checking.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**